production.yml

```diff
@@ -18,26 +18,16 @@ spec:
       name: apr
     spec:
       containers:
-      - image: 585031190124.dkr.ecr.us-east-1.amazonaws.com/apr:6337b32ce71e38831b1d680436a5e3e128b6fe82
+      - image: 585031190124.dkr.ecr.us-east-1.amazonaws.com/apr:latest
         imagePullPolicy: Always
         name: apr
         env:
-        - name: BASIC_AUTH_USER
```

Comment:
These ENV vars seem to be only needed at compile time.  Changing them at runtime has no effect